servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/WriteStreamSubscriber.java

```diff
@@ -519,6 +520,8 @@ private void terminateSubscriber(@Nullable Throwable cause) {
                 Throwable enrichedCause = enrichProtocolError.apply(cause);
                 assignConnectionError(channel, enrichedCause);
                 enrichedCause = !written ? new AbortedFirstWriteException(enrichedCause) : enrichedCause;
+                LOGGER.debug("{} Terminate subscriber with an error, state: {}", channel, Integer.toString(state, 2),
```

Comment:
error termination isn't expected to be common, but do we want todo `Integer.toString(state, 2)` unconditionally?